client/scss/components/_form-side.scss

```diff
@@ -49,12 +59,38 @@
     }
   }
 
+  &__resize-grip-container {
+    @apply w-absolute w-place-items-center w-hidden md:w-flex w-z-10 w-left-[-22px];
+
+    height: calc(100% - theme('spacing.8'));
+  }
+
+  &__resize-grip {
+    @apply w-text-primary hover:w-text-primary-200 w-border w-border-transparent w-rounded w-bg-white w-p-3 w-hidden w-touch-pinch-zoom w-cursor-ew-resize;
```

Comment:
What do you think of shifting this one pixel to the right? It feels to me slightly better aligned this way, though with an even width aiming for the exact middle is hopeless.

Reply:
I think I previously used `w-left-[-21px]` (so, one pixel to the right) and felt it's slightly better aligned with `-22px`. Shall we go with `-21.5px`? Not sure if half a pixel make sense, but [it seems valid](https://stackoverflow.com/questions/9080633/can-a-css-pixel-be-a-fraction) 😛

Reply:
Let’s ask a design person what they think!